src/main/webapp/user-page.html

```diff
@@ -115,6 +116,11 @@ <h2 class="section-heading text-uppercase">My posts:</h2>
       </div>
     </section>
 
+    <!-- Places I've been -->
```

Comment:
Could be nice to have a header or something also say "Places I've been" so that its not just a random map, just a suggestion